q2_types/feature_data/_transformer.py

```diff
@@ -319,3 +353,13 @@ def _19(data: AlignedDNAIterator) -> AlignedDNAFASTAFormat:
     ff = AlignedDNAFASTAFormat()
     skbio.io.write(iter(data), format='fasta', into=str(ff))
     return ff
+
+
+@plugin.register_transformer
+def _32(ff: AlignedDNAFASTAFormat) -> pd.Series:
```

Comment:
Can you remove this transformer since it isn't needed right now? I'd rather keep the transformers limited to what's currently needed, and add missing transformers when a use-case comes up. Since transformers are likely to change in significant ways in the future it'll be easier to migrate if there's fewer out in the wild (some of the transformer overhaul involves rethinking how `typing` works, and adding transformer transitivity).